CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)
 
 ## [Unreleased]
+### Added
+- Added many checks for DNS zone (@yuri-zubov sponsored by Actility, https://www.actility.com)
+
+### Breaking Changes
+- Dropping ruby `< 2.1` support (@yuri-zubov)
```

Comment:
in the gemspec you say it must be `2.2+` but here you have `< 2.1` I think this should be `< 2.2` here.